### ISSUE TITLE
Error in IKVM PackageListAttribute constructor when calling AutoDetectParser

### ISSUE THREAD

**Issue by alangreeneweb** [KevM/tikaondotnet#24]
Since the latest update through nuget for TikaOnDotNet and IKVM, I have started getting errors.  I also get the same error when running the Test Project included with TikaOnDotNet.
I was wondering if you have run into this problem.
 
After AutoDetectParser is called, I was getting this error:
 InnerException {"Method not found: 'Void IKVM.Attributes.PackageListAttribute..ctor(System.String[][])'."}             System.Exception {System.MissingMethodException}
 
I tried with both the 7.x and 8.x versions of IKVM.
 
I grabbed the IKVMC compiler and compiled the latest tika-app jar file which gives a slightly different constructor error:
 InnerException {"Method not found: 'Void IKVM.Attributes.PackageListAttribute..ctor(System.String[])'."}              {System.MissingMethodException}
 
 I was wondering if you had any idea on where I should head on this one.  I think I’ll pull the IKVM source and consider adding a matching constructor, but that seems like I’m just stabbing around in the dark and could be missing something easy.

**Comment by KevM** [comment]
I just cloned, built, and tested with no problem. Are you on a fancy new version of .Net or running Windows 10 or something "exotic"?

**Comment by alangreeneweb** [comment]
VS 2013.

Windows 8.1.

I switched to IKVM 7.4 which has the PackageListAttribute constructor that takes a single dimension string array and it’s working now.  I will remove the issue.  I added the same issue for the IKVM guys.

 

So here’s where I’m at, with it working.

1.       Your latest TikaOnDotNet

2.       IKVM 7.4.xxx

3.       Tika-app_1.8.jar compiled with ikvmc, using technically your MySystemClassLoader, but it’s in my dll, so I referenced my dll when doing the ikvmc compile.
